grpc-codegen.sh

```diff
@@ -22,8 +22,10 @@ trap clean ERR
 if [ ! -d "$PROTO_PATH" ]; then
     mkdir -p "$PROTO_PATH" && (
       cd "$PROTO_PATH"
-      curl -s https://digitalassetsdk.bintray.com/DigitalAssetSDK/com/digitalasset/ledger-api-protos/"$SDK_VERSION"/ledger-api-protos-"$SDK_VERSION".tar.gz | tar xz --strip-components 2
-      curl -s https://digitalassetsdk.bintray.com/DigitalAssetSDK/com/digitalasset/daml-lf-archive-protos/"$SDK_VERSION"/daml-lf-archive-protos-"$SDK_VERSION".tar.gz | tar xz --strip-components 3
+      curl -s https://repo1.maven.org/maven2/com/digitalasset/ledger-api-protos/$SDK_VERSION/ledger-api-protos-$SDK_VERSION.tar.gz | tar xz --strip-components 2
+      curl -s https://digitalassetsdk.bintray.com/DigitalAssetSDK/com/digitalasset/daml-lf-1.6-archive-proto/"$SDK_VERSION"/daml-lf-1.6-archive-proto-"$SDK_VERSION".tar.gz | tar xz
```

Comment:
Why do we use 1.6?

Reply:
@bame-da Do we want to move to 1.7 or ask users to work with 1.6?

Reply:
If 1.7 is not fully supported, let's stick to 1.6 for now.